batch/extension-jberet/src/main/java/org/wildfly/extension/batch/jberet/BatchSubsystemDefinition.java

```diff
@@ -84,7 +84,7 @@
     static final SimpleAttributeDefinition DEFAULT_THREAD_POOL = SimpleAttributeDefinitionBuilder.create("default-thread-pool", ModelType.STRING, true)
             .setAllowExpression(false)
             .setAttributeGroup("environment")
-            .setAttributeMarshaller(NameAttributeMarshaller.INSTANCE)
+            .setAttributeMarshaller(NameAttributeMarshaller.INSTANCE) //todo this looks like it is missing capability reference
```

Comment:
Right. Service != capability though :)

Yes, I started that, but just didn't have time. It wouldn't be all that difficult, just time consuming. Really the main comment is I'd prefer not add a TODO for something that may or may not be fixable due to the how the thread pool resources work.